Makefile

```diff
@@ -442,7 +442,7 @@ clean-helm:
 	@rm -rf $(HELM)
 
 "$(GOBIN)/kind":
-	go install sigs.k8s.io/kind@$(KIND_VERSION)
+	CGO_ENABLED=0 go install sigs.k8s.io/kind@$(KIND_VERSION)
```

Comment:
TIL CGO_ENABLED=0 is only default on OSes without a C toolchain installed.

What error does this fix here?

Reply:
Without this the `kind` binary cannot be invoked inside the `kind-e2e` container: https://oss.gprow.dev/view/gs/oss-prow-build-kpt-config-sync/pr-logs/pull/GoogleContainerTools_kpt-config-sync/1336/kpt-config-sync-presubmit-e2e-multi-repo-test-group1/1812970304033525760

Reply:
Can you add a comment. Something like "Build kind with CGO disabled so it can be used in containers without C installed.

Reply:
done